src/account/AccountInfo.js

```diff
@@ -31,8 +31,11 @@ import PublicKey from "../PublicKey.js";
 import LedgerId from "../LedgerId.js";
 
 /**
+ * @deprecated - no longer supported
  * @typedef {import("./HbarAllowance.js").default} HbarAllowance
+ * @deprecated - no longer supported
  * @typedef {import("./TokenAllowance.js").default} TokenAllowance
+ * @deprecated - no longer supported
```

Comment:
`@deprecated` should only be added above the actual fields

Reply:
updated